pkg/agent/datamodel/sig_config.go

```diff
@@ -495,55 +526,111 @@ var (
+		ResourceGroup: AKSAzureLinuxResourceGroup,
+		Gallery:       AKSAzureLinuxGalleryName,
+		Definition:    "V2gen2",
+		Version:       "202309.11.0",
+	}
+
 	SIGCBLMarinerV2Gen1FIPSImageConfigTemplate = SigImageConfigTemplate{
 		ResourceGroup: AKSCBLMarinerResourceGroup,
 		Gallery:       AKSCBLMarinerGalleryName,
 		Definition:    "V2fips",
 		Version:       LinuxSIGImageVersion,
 	}
 
+	SIGAzureLinuxV2Gen1FIPSImageConfigTemplate = SigImageConfigTemplate{
+		ResourceGroup: AKSAzureLinuxResourceGroup,
+		Gallery:       AKSAzureLinuxGalleryName,
+		Definition:    "V2fips",
+		Version:       "202309.11.0",
+	}
+
 	SIGCBLMarinerV2Gen2FIPSImageConfigTemplate = SigImageConfigTemplate{
 		ResourceGroup: AKSCBLMarinerResourceGroup,
 		Gallery:       AKSCBLMarinerGalleryName,
 		Definition:    "V2gen2fips",
 		Version:       LinuxSIGImageVersion,
 	}
 
+	SIGAzureLinuxV2Gen2FIPSImageConfigTemplate = SigImageConfigTemplate{
+		ResourceGroup: AKSAzureLinuxResourceGroup,
+		Gallery:       AKSAzureLinuxGalleryName,
+		Definition:    "V2gen2fips",
+		Version:       "202309.11.0",
+	}
+
 	SIGCBLMarinerV2KataImageConfigTemplate = SigImageConfigTemplate{
 		ResourceGroup: AKSCBLMarinerResourceGroup,
 		Gallery:       AKSCBLMarinerGalleryName,
 		Definition:    "V2katagen2",
 		Version:       LinuxSIGImageVersion,
 	}
 
+	SIGAzureLinuxV2KataImageConfigTemplate = SigImageConfigTemplate{
+		ResourceGroup: AKSAzureLinuxResourceGroup,
+		Gallery:       AKSAzureLinuxGalleryName,
+		Definition:    "V2katagen2",
+		Version:       "202309.11.0",
+	}
+
 	SIGCBLMarinerV2Arm64ImageConfigTemplate = SigImageConfigTemplate{
 		ResourceGroup: AKSCBLMarinerResourceGroup,
 		Gallery:       AKSCBLMarinerGalleryName,
 		Definition:    "V2gen2arm64",
 		Version:       LinuxSIGImageVersion,
 	}
 
+	SIGAzureLinuxV2Arm64ImageConfigTemplate = SigImageConfigTemplate{
+		ResourceGroup: AKSAzureLinuxResourceGroup,
+		Gallery:       AKSAzureLinuxGalleryName,
+		Definition:    "V2gen2arm64",
+		Version:       "202309.11.0",
+	}
+
 	SIGCBLMarinerV2TLImageConfigTemplate = SigImageConfigTemplate{
 		ResourceGroup: AKSCBLMarinerResourceGroup,
 		Gallery:       AKSCBLMarinerGalleryName,
 		Definition:    "V2gen2TL",
 		Version:       LinuxSIGImageVersion,
 	}
 
+	SIGAzureLinuxV2TLImageConfigTemplate = SigImageConfigTemplate{
+		ResourceGroup: AKSAzureLinuxResourceGroup,
+		Gallery:       AKSAzureLinuxGalleryName,
+		Definition:    "V2gen2TL",
+		Version:       "202309.11.0",
+	}
+
 	SIGCBLMarinerV2KataGen2TLImageConfigTemplate = SigImageConfigTemplate{
 		ResourceGroup: AKSCBLMarinerResourceGroup,
 		Gallery:       AKSCBLMarinerGalleryName,
 		Definition:    "V2katagen2TL",
 		Version:       CBLMarinerV2KataGen2TLSIGImageVersion,
 	}
 
+	SIGAzureLinuxV2KataGen2TLImageConfigTemplate = SigImageConfigTemplate{
+		ResourceGroup: AKSAzureLinuxResourceGroup,
+		Gallery:       AKSAzureLinuxGalleryName,
+		Definition:    "V2katagen2TL",
+		Version:       CBLMarinerV2KataGen2TLSIGImageVersion,
```

Comment:
we didnt actually release katagen2TL image for cgroupsv2, maybe we can remove this?

Reply:
Fixed